### PR TITLE
Create masterfile for available datasets

### LINKED ISSUE #58: Create masterfile for available datasets
Need a masterfile to check available datasets and metadata for download API.

### PR DESCRIPTION
**Pull request by mauriciogtec**
Adds a datamaster class that lists available datasets and facilitates accessing information about the url, metadata, etc.
The main functionality is in `datasets/datamaster.py`.
There are two key files `datasets/masterfile.csv` and `datasets/collectionts.csv`.
Fixes #58

### CODE REVIEW

**Review by atrisovic — COMMENTED** [review]
I was thinking we can keep all that info in the DV metadata :smile: and they'd be accessible with something like `spacebench list` but I'm not sure what are cons and pros of having a physical file in the package

**Comment by atrisovic** [comment]
maybe the master file csvs can also be on DV 🤔

File: tests/test_datamaster.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import unittest
+
+import pandas as pd
+
+from spacebench.datasets.datamaster import DataMaster
+
+
+class TestDataMaster(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
```
Comment (Naeemkh) [inline tests/test_datamaster.py:9]: You do not need to initialize the class. Please use `setUp` and `tearDown` functions. Here is an example:

```python
import unittest

class WidgetTestCase(unittest.TestCase):
    def setUp(self):
        self.widget = Widget('The widget')

    # Your tests go here

    def tearDown(self):
        self.widget.dispose()
```

**Review by Naeemkh — CHANGES_REQUESTED** [review]

**Comment by mauriciogtec** [comment]
@atrisovic I am fine if the data is on DV later on if we can write code that accesses it and checks against it every time. But I don't want to hard-code logic about what dataset to download anymore (as it is now) based on the patterns of existing files, since that will scale.

File: tests/test_datamaster.py (hunk 1 above)
Comment (mauriciogtec) [reply tests/test_datamaster.py:9]: fixed

**Pull request merged**